### PR TITLE
Scene1, Scene4 Added

### PR DESCRIPTION
**Pull request by giraffeJ**
Scene1 : GoldenBug Game
Scene4 : Ending scene draft

### CODE REVIEW

File: Assets/Scene0/ExitHandler.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class ExitHandler : MonoBehaviour{
 
   }
 
+
```
Comment (younghk) [inline Assets/Scene0/ExitHandler.cs:21]: blank line check plz...

File: Assets/Scene0/LoadingBar.cs [hunk 2]
```diff
@@ -47,6 +47,7 @@ void Update()
   }
   public async void initAmount()
   {
+      Debug.Log("clicked");
```
Comment (younghk) [inline Assets/Scene0/LoadingBar.cs:50]: plz remove debug log when release this project...

**Review by JeonYongHyun — APPROVED** [review]
수고했다

File: Assets/Scene0/ExitHandler.cs (hunk 1 above)
Comment (giraffeJ) [reply Assets/Scene0/ExitHandler.cs:21]: Sorry

File: Assets/Scene0/LoadingBar.cs (hunk 2 above)
Comment (giraffeJ) [reply Assets/Scene0/LoadingBar.cs:50]: I believe that you would do that :)

**Pull request merged**